test/e2e/suite_test.go

```diff
@@ -154,6 +154,8 @@ var _ = BeforeSuite(func() {
 					"global.cattle.psp.enabled=false",
 					"--set", fmt.Sprintf("hostname=%s.%s", e2eCfg.ExternalIP, e2eCfg.MagicDNS),
 					"--create-namespace",
+					"--devel",
+					"--set rancherImageTag=v2.9-head",
```

Comment:
same comment as in EKS operator: https://github.com/rancher/eks-operator/pull/376#discussion_r1502337953

Reply:
fixed